addons/web/views/report_templates.xml

```diff
@@ -4,18 +4,13 @@
 
     <template id="report_assets_common">
         <t t-call="web._assets_helpers">
-            <!-- FIXME web_editor ? -->
           <link rel="stylesheet" type="text/scss" href="/web/static/src/scss/bootstrap_overridden_report.scss"/>
-          <link rel="stylesheet" type="text/scss" href="/web_editor/static/src/scss/bootstrap_overridden.scss"/>
         </t>
 
         <t t-call="web._assets_bootstrap"/>
 
         <link href="/base/static/src/css/description.css" rel="stylesheet" type="text/css"/>
 
-        <!-- FIXME web_editor ? -->
```

Comment:
unnecessary changes, even kinda dangerous ??

Reply:
@kebeclibre Well, it kinda is yes. Problem is, with those lines, there is a scss problem in tests. It says that a variable is missing (which is true, the files here requires data from other non-imported web editor files). And those lines are called again in the webeditor.xml. I mentionned it here in the discord: https://discordapp.com/channels/678381219515465750/687338039717920792/742996732450963516 and following the answers I tried to remove it. No test break so technically we're safe :D (just kidding of course)

Reply:
Ho, and it's not one test that fails. It's more like for every tests, it screams there is a problem in the scss compilation because of a missing var

Reply:
fair enough :D